api/src/Entity/TopBook.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use ApiPlatform\Core\Annotation\ApiProperty;
+use ApiPlatform\Core\Annotation\ApiResource;
+
+/**
+ * This entity represents a "most borrowed book" in a given a given French library.
+ * It is loaded from a CSV file. It's purpose it to show how to expose data coming
+ * from external data sources through API Platform.
+ *
+ * @see https://www.data.gouv.fr/fr/datasets/top-100-romans-adultes-de-science-fiction-et-fantastiques/
+ * @see https://www.data.gouv.fr/fr/datasets/r/a2b09081-6cc4-4fdb-bd73-35484014c89c
+ * @see /data/top-100-novel-sci-fi-fr.csv
+ *
+ * @ApiResource(
+ *   collectionOperations={"get"},
+ *   itemOperations={"get"}
+ * )
+ */
+class TopBook
+{
+    /**
+     * This is a fake ID so we can test the get "item-by-id" operation.
+     *
+     * @ApiProperty(identifier=true)
+     */
+    private ?int $id = null;
+
+    /**
+     * Title of the book.
```

Comment:
Useless documentation: we're in a TopBook, the property is called title, we can easily guess this is the "Title of the book" :D (same for other properties)

Reply:
OK for title and author, but it's not the same for other properties. For example, for the place property it's not obvious what it is exactly (same for part). For the borrow count, it gives an information the time range the count is referring to.

Reply:
Yep, you can keep the documentation if it's useful